sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/aio/_patch.py

```diff
@@ -167,8 +162,7 @@ def from_connection_string(cls, connection_string, hub, **kwargs):
         return cls(hub=hub, credential=credential, **kwargs)
 
     @distributed_trace_async
-    async def get_client_access_token(self, **kwargs):
-        # type: (Any) -> JSONType
+    async def get_client_access_token(self, **kwargs: Any) -> JSONType:
```

Comment:
In the generated code we have:
```python
    async def get_client_access_token(
        self,
        *,
        user_id: Optional[str] = None,
        roles: Optional[List[str]] = None,
        minutes_to_expire: Optional[int] = 60,
        **kwargs: Any
    ) -> JSONType:
```

Reply:
oh good point, we should have it keyword only in the aio patch.py file too. thanks!